src/pages/patterns/global-header-pattern/index.mdx

```diff
@@ -0,0 +1,297 @@
+| ----------------- | -------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| **Global region** | Log in, notifications, profile access, dashboard link, setting time, billing information, user permissions, switching from one product to another. | Hamburger menu to toggle local nav. Show/hide application navigation, document thumbnails. Consistent placement of local actions in common between products for example: new file, save, cancel. |
+| **Local region**  | Global settings, user rights management, notification preferences.                                                                                 | Commenting in a text doc, navigating from one peer resource to another (for example from one container, data base, or document to another), selecting modalities within your product.            |
+
+</Column>
+</Row>
+
+## Configurations
+
+The UI shell is configurable to let a product or platform choose which shell components and configurations match their user and information requirements. Simple products have the flexibility of using a header or side panel and more complex products require a combination of UI shell components to accommodate the depth of their navigation.
+
+![Example the header and left nav being used for the same IA.](images/configurations.png)
+
+<Caption>
+  A simple product navigation can be achieved with either a header or side panel
+  component.
+</Caption>
+
+### Header only
+
+The UI Shell header can be used as standalone navigation for your UI when a small number of main sections do not require a secondary navigation. The header provides a place for a site title, navigation links and dropdowns, and header icons. The header is globally persistent and will always be in view as the user navigates around the site.
+
+This configuration gives more horizontal space for the page's content and has limited room for navigation items in the header. This configuration also does not lend itself to sub-menus that need to remain open as they overlay and interfere with the page's content. Use a left panel in combination with the header if your navigation requires a sub-menu to remain open.
+
+<Row>
+<Column colLg={8}>
+
+![Example of a header as the global navigation](images/header-only-example.png)
+
+<Caption>An example of a header-only navigation.</Caption>
+
+</Column>
+</Row>
+
+### Header with left panel
+
+The left panel allows for more navigational items to stack vertically and an additional level of hierarchy when paired with header navigation. Compared to a head-only site navigation, this arrangement means sub-menus can remain open without overlaying or interfering with the page's content.
```

Comment:
head-only should be header-only?

Reply:
>Compared to a head-only site navigation, this arrangement means sub-menus can remain open without overlaying or interfering with the page's content.

should be

With this arrangement sub-menus can remain open without overlaying or interfering with the page's content.